tests/console/hostname.pm

```diff
@@ -22,6 +22,7 @@ sub run() {
 
     script_run "hostname";
     assert_screen("hostname-$hostname");
+    assert_script_run "systemctl -q is-active network.service && systemctl restart network.service";
```

Comment:
But isn't your HA HDD made after this is done..so how does that help the dyndns? or am I missing something